### DISCUSSION TITLE
Number of results returned?

### DISCUSSION THREAD

**Discussion by PowershellScripter** [benbusby/whoogle-search#412] [Q&A] [answered]
Is there any way to change the number of results returned? Like, increasing to 100, both for the search results and image search results?

**Comment by DUOLabs333** [comment]
This has been asked in #466

**Comment by benbusby** [comment, marked as answer]
Number of results is configurable as of #486 using the `WHOOGLE_RESULTS_PER_PAGE` environment variable.
